bundles/org.openhab.binding.smhi/README.md

```diff
@@ -0,0 +1,143 @@
+# Smhi Binding
+
+This binding gets hourly and daily forecast from SMHI - the Swedish Meteorological and Hydrological Institute. It can't 
```

Comment:
The new line is on a typical place. Better put the different sentences on their own lines that is easier if the file even has to updated/reviewed later on.